src/components/ListRow/index.js

```diff
@@ -0,0 +1,145 @@
+import React from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import { spacing, colors } from "../../theme";
+
+import Button from "../Button";
+import { Row, Column } from "../Grid";
+import { PrimaryText, SecondaryText, BoldText, Link } from "../Text";
+import { OverflowIcon } from "../Icons";
+import { mediumAndUp, largeAndUp } from "../../theme/mediaQueries";
+
+const variants = ["standard", "withLink"];
+
+const ListContainer = styled.div`
+  padding: ${spacing.moderate} 0
+    ${props => (props.variant === "withLink" ? spacing.cozy : "12px")} 12px;
+  ${mediumAndUp`
+    padding: 18px 0
+      ${props => (props.variant === "withLink" ? spacing.cozy : "18px")}
+      ${props => (props.onExpand ? 0 : "12px")};
+  `};
+`;
+
+const ListLink = styled.a`
+  cursor: pointer;
+`;
+
+const ListRow = styled(Row)`
+  display: flex;
+  align-items: center;
+`;
+
+const DesktopOnlyColumn = styled(Column)`
+  display: none;
+  ${mediumAndUp`
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  `};
+`;
+
+const MobileOnlyColumn = styled(Column)`
+  ${mediumAndUp`
+  display: none;
+  `};
+`;
+
+const SubTitleColumn = styled(Column)`
+  display: none;
+  ${largeAndUp`
+  display: flex;
+    justify-content: center;
+    align-items: center;
+  `};
+`;
+
+const ListItem = ({
+  title,
+  subTitle,
+  dateTitle,
+  dateSubTitle,
+  buttonText,
+  variant,
+  linkTitle,
+  linkUrl,
+  coloredDate,
+  onClick,
+  onOverflowClick
+}) => (
+  <ListContainer variant={variant} onExpand={onClick}>
+    <ListLink onClick={onClick}>
+      <ListRow>
+        <Column small={3} medium={1.8} large={1.3} xLarge={1}>
```

Comment:
In order for column sizes to render properly only whole numbers should be provided. Same through the entire PR